src/webgpu/web_platform/reftests/canvas_size_different_with_back_buffer_size.html.ts

```diff
@@ -163,19 +170,48 @@ export function run() {
       ]);
     }
 
-    // Test canvas size, back buffer size and CSS size are different
+    // Test back buffer size < canvas size < CSS size
     {
-      const back_buffer_width = cvs_back_buffer_css_different_size.width / 2;
-      const back_buffer_height = cvs_back_buffer_css_different_size.height / 2;
-      const ctx = cvs_back_buffer_css_different_size.getContext('gpupresent');
+      const back_buffer_width = back_buffer_smaller_than_cvs_and_css.width / 2;
+      const back_buffer_height = back_buffer_smaller_than_cvs_and_css.height / 2;
+      const ctx = back_buffer_smaller_than_cvs_and_css.getContext('gpupresent');
       assert(ctx !== null);
 
-      updateWebGPUBackBuffer(ctx, back_buffer_width, back_buffer_height, [
-        [red, red, green],
-        [red, red, green],
-        [blue, blue, yellow],
-        [blue, blue, yellow],
-      ]);
+      updateWebGPUBackBuffer(
+        back_buffer_smaller_than_cvs_and_css,
+        ctx,
+        [back_buffer_width, back_buffer_height],
+        [
+          [red, red, green],
+          [red, red, green],
+          [blue, blue, yellow],
+          [blue, blue, yellow],
+        ]
+      );
+    }
+
+    // Test canvas size < back buffer size < CSS size
+    {
+      const back_buffer_width = cvs_smaller_than_back_buffer_and_css.width * 2;
+      const back_buffer_height = cvs_smaller_than_back_buffer_and_css.height * 2;
+      const ctx = cvs_smaller_than_back_buffer_and_css.getContext('gpupresent');
+      assert(ctx !== null);
+
+      updateWebGPUBackBuffer(
+        cvs_smaller_than_back_buffer_and_css,
+        ctx,
+        [back_buffer_width, back_buffer_height],
+        [
+          [red, red, red, red, green, green],
+          [red, red, red, red, green, green],
+          [red, red, red, red, green, green],
+          [red, red, red, red, green, green],
+          [blue, blue, blue, blue, yellow, yellow],
+          [blue, blue, blue, blue, yellow, yellow],
+          [blue, blue, blue, blue, yellow, yellow],
+          [blue, blue, blue, blue, yellow, yellow],
+        ]
```

Comment:
nit: I think we wouldn't be able to tell if this were downsampled to 3x4 before being upsampled to 12x16

Reply:
Fixed by adding one pixel

Reply:
That's clever, thanks and got the skill.